src/API/Features/Health/HealthController.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace API.Features.Health
+{
+    using Infrastructure;
+    using Microsoft.AspNetCore.Mvc;
+    using System.Reflection;
+
```

Comment:
@sungcheng, please note coding style where the ```using``` section is inside the namespace.